packages/core/src/RenderingEngine/vtkClasses/vtkStreamingOpenGLVolumeMapper.js

```diff
@@ -246,24 +246,12 @@ function vtkStreamingOpenGLVolumeMapper(publicAPI, model) {
   publicAPI.setCameraShaderParameters = (cellBO, ren, actor) => {
     const program = cellBO.getProgram();
     const cam = model.openGLCamera.getRenderable();
-    const blendMode = actor.getMapper().getBlendMode();
-    const slabThickness = cam.getSlabThickness();
     const crange = cam.getClippingRange();
-
-    const defaultSlabThickness = null;
-    const cameraMidpoint = (crange[1] + crange[0]) * 0.5;
-    // if not equal to tiny slab thickness (i.e. it is defined as an actual
-    // intended value), use slab thickness instead of clipping range
-    if (
-      blendMode !== BlendMode.COMPOSITE_BLEND &&
-      slabThickness !== defaultSlabThickness
-    ) {
-      crange[0] = cameraMidpoint - slabThickness;
-      crange[1] = cameraMidpoint + slabThickness;
-      cam.setSlabThicknessActive(true);
-    } else {
-      cam.setSlabThicknessActive(false);
-    }
+    // NOTE: the actual slab thickness clipping is done with clipping planes,
+    // but here we still need to have these values here, otherwise
+    // the rendering will be clipped before the clipping planes.
+    // The clipping range is set in the Viewports (in resetCamera) and
+    // have values: [0.01, model.distance * 2]
```

Comment:
I think this comment is invalid, "here we still need to have these values", which values?